metabase/metabase
# create static scatter plot with echarts

Closes https://github.com/metabase/metabase/issues/35800

### Description

Renders the basic series for the static scatter plot implemented with echarts. I also added many storybook examples for different cases that will be implemented in later PRs.

### How to verify

1. Create a basic scatter plot
2. Add to a dashboard
3. Send a subscription
4. Confirm it renders correctly

### Demo

![Screenshot 2023-12-14 at 2.33.05 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/3f71c67c-dd8d-4bc7-b7dd-a9946a297f29.png)

### Checklist
